desktop/core/src/desktop/js/reactComponents/MetricsComponent/MetricsTable.tsx

```diff
@@ -0,0 +1,104 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// 'License'); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react';
+import Table from 'cuix/dist/components/Table/Table';
+import type { ColumnType } from 'antd/es/table';
+import './MetricsComponent.scss';
+
+interface MetricsValue {
+  value: number;
+}
+interface MetricsTime {
```

Comment:
since we are handling these metrics explicitly here already and showing them as labels later, why not also transform them to have proper labels, i.e. remove the snake_case and add i18n.